semgrep-core/tests/Test.ml

```diff
@@ -94,7 +94,13 @@ let regression_tests_for_lang files lang =
     let rule = { R.
       id = "unit testing"; pattern; message = ""; severity = R.Error; 
       languages = [lang] } in
-    let equiv = [] in
+    let equiv = 
+     (* Python == is not the same than !(==) *)
+     if lang <> Lang.Python
```

Comment:
FWIW, I think `basic_equivalences.yml` already excludes Python from the rewrite. Do we still need the language check here?

Reply:
Oh, yeah, you're right! Actually I just realized the equivalence code does not respect the languages[] field
in the equivalence file ... I'll fix that in another diff.